package/MDAnalysis/lib/formats/src/xdrfile.c

```diff
@@ -44,6 +44,8 @@
 #  include <inttypes.h>
 #endif
 
+#include <sys/types.h>
+#include <unistd.h>
```

Comment:
@tylerjereddy this might enable usage of `off_t` on windows as well.  I have to include the headers since the `off_t` type is not officially part of the c99 standard.